dojo/models.py

```diff
@@ -2943,7 +2949,7 @@ def save(self, dedupe_option=True, rules_option=True, product_grading_option=Tru
             finding_helper.post_process_finding_save(self, dedupe_option=dedupe_option, rules_option=rules_option, product_grading_option=product_grading_option,
                 issue_updater_option=issue_updater_option, push_to_jira=push_to_jira, user=user, *args, **kwargs)
         else:
-            logger.debug('no options selected that require finding post processing')
+            logger.debug('no options selected that require finding post processing')        
```

Comment:
Flake8 is cranky in various parts of this file that you did not touch, but totally missed this one that is a violation. Something must have been bugged out when it ran. Please remove trailing whitespaces